keyboards/zinc/keymaps/toshi0383/rules.mk

```diff
@@ -1,11 +1,12 @@
 # Build Options
-#   change yes to no to disable
+#   change to "no" to disable the options, or define them in the Makefile in
+#   the appropriate keymap folder that will get included automatically
 #
-BOOTMAGIC_ENABLE = no       # Virtual DIP switch configuration
-MOUSEKEY_ENABLE = no        # Mouse keys
-EXTRAKEY_ENABLE = yes       # Audio control and System control
-CONSOLE_ENABLE = no         # Console for debug
-COMMAND_ENABLE = no         # Commands for debug and configuration
+BOOTMAGIC_ENABLE = no       # Virtual DIP switch configuration(+1000)
```

Comment:
```suggestion
BOOTMAGIC_ENABLE = no       # Virtual DIP switch configuration
```

Reply:
I checked the latest version of qmk, so many '# Virtual DIP switch configuration(+1000)' exist...
864 results in 864 files 
Should we change?

Reply:
Yes, see https://github.com/qmk/qmk_firmware/pull/6947